crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -1916,6 +1920,7 @@ impl SuiTransactionEffects {
         Ok(Self {
             status: effect.status.into(),
             gas_used: effect.gas_used.into(),
+            modified_at_versions: effect.modified_at_versions,
```

Comment:
Separate issue, but wondering why the existing `modified_at_versions` doesn't include the input object digest. This is important info if you want to replay the tx by reading the input values from an untrusted store.

Reply:
Originally `modified_at_versions` was included to support reverting state updates, which assume a trusted store (the validator's own), so that's the reason for the omission.  We chatted about this question in a recent meeting, and while this isn't necessary to support querying object history, it will be for replaying a transaction against an untrusted  store -- if that's something that we support, we should add the digest.